R/uvhydrograph-render.R

```diff
@@ -72,6 +72,11 @@ createPrimaryPlot <- function(data, month){
         plot_object <- extendYaxisLimits(plot_object, primaryStyles[[err]])
       }
       
+      which_rect <- grep('rect', names(primaryStyles))
```

Comment:
if the implementation of addHeight I commented above works then this and the for loop after it can be deleted